### ISSUE TITLE
Errors in Help

### ISSUE THREAD

**Issue by pacalet** [rarsm/rars#98]
The RISCV tab of the Help window of RARS 1.6 is split in 2 halves. The text in the top half (with green background) assumes 16 bits immediate operands (-32768 to 32767) while most RISCV immediate operands are only 12 bits long (-2048 to 2047).

**Comment by privat** [comment]
Yeah, this should be fixed. Moreover, the "green part" is not that much discoverable.
